Source/Meadow.Foundation.Peripherals/Sensors.Motion.Lsm303agr/Driver/Lsm303agr.Registers.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Meadow.Foundation.Sensors.Accelerometers
+{
+    public partial class Lsm303agr
```

Comment:
Any particular reason these registers are public?

Reply:
its a partial class, but the registers are not public

Reply:
yeah they're not public - partial public class but the fields are private